src/ui/ProfileBar.module.css

```
@@ -101,32 +103,41 @@
+  display: inline-flex;
+  align-items: center;
+  gap: 3px;
+  flex-shrink: 0;
+  opacity: 0.85;
+}
+
 .dirtyDot {
   width: 6px;
   height: 6px;
   border-radius: 50%;
   background: currentColor;
```

Stylelint (17.7.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)